protobuf/pg_query.proto

```diff
@@ -4108,3 +4108,44 @@ enum Token {
   MODE_PLPGSQL_ASSIGN3 = 777;
   UMINUS = 778;
 }
+
+
+message SummaryResult {
+  enum Context {
+    None = 0;
+    Select = 1;
+    DML = 2;
+    DDL = 3;
+    Call = 4;
+  }
+
+  message Table {
+    string table = 1;
+    Context context = 2;
+  }
+  // tables: Vec<(String, Context)>
+  repeated Table tables = 1;
+
+  // aliases: HashMap<String, String>
+  map<string, string> aliases = 2;
```

Comment:
For clarity, the alias value here is the table name (i.e. `schema.table` or just `table` depending on how its written in the query).